eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -49,9 +52,63 @@ jobs:
     BuildInParallel: ${{ parameters.BuildInParallel }}
     TimeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
     Location: ${{ parameters.Location }}
-    SubscriptionConfiguration: ${{ parameters.SubscriptionConfiguration }}
     ServiceDirectory: ${{ parameters.ServiceDirectory }}
     TestSetupSteps: ${{ parameters.TestSetupSteps }}
+    CloudConfigurations:
+      ${{ if contains(parameters.Clouds, 'AzureCloud') }}:
```

Comment:
We should decide if we want to include our other subscription configurations as part of these. Things like canary and preview.

Reply:
From a pipeline authoring/simplicity perspective, I'm leaning towards representing the subscription configurations within their own cloud configs, as opposed to injecting them in via boolean parameters and if conditionals. This way as we add more configurations, it just becomes an entry to the cloud config array, meaning all combinations get captured explicitly by name.

Right now there are a few sdk configs consuming the SubscriptionConfiguration parameter directly, so it shouldn't be hard to move one way or the other as far as keeping sdk tests working.